dependencies.yaml

```diff
@@ -336,3 +344,15 @@ dependencies:
           - matrix: # All CUDA 11 versions
             packages:
               - cuda-python>=11.7.1,<12.0a0
+  test_python_legate:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - *dask
+          - distributed>=2022.05.2
```

Comment:
This is probably from an improper merge conflict resolution. I think we can delete it.
```suggestion
```